### DISCUSSION TITLE
program freeze when using 1micro second trajectory

### DISCUSSION THREAD

**Discussion by mayank-kohli** [Valdes-Tresanco-MS/gmx_MMPBSA#212] [Q&A] [answered]
when I am using a gromacs trajectory of 1 micro second the program freezes during this step "Cleaning normal complex trajectories..."
and the file size of "COM_traj_0.xtc" stuck at 342 mb and it wont response after that.
But when I did for the 100ns trajectory I didn't faced any error everything ran smoothly. 

Does anyone have any idea where I might be going wrong?

**Comment by marioernestovaldes** [comment]
a possible workaround for a problem like this is to provide a "clean" trajectory, that is, a trajectory containing only the elements you are interested in, and then set solvated_trajectory=1 in the &general namelist of the mmpbsa.in file... have a go with this a see how it goes

**Reply by mayank-kohli** [reply]
I have tried this also by providing only the receptor and ligand, still I am facing the same thing.
when I increased the interval value to 10, it worked. But calculating for only 10000 frames is not relevant.

**Comment by marioernestovaldes** [comment, marked as answer]
answered [here](https://github.com/Valdes-Tresanco-MS/gmx_MMPBSA/issues/213)
